engine/src/java/com/runrev/android/Engine.java

```diff
@@ -3443,6 +3458,11 @@ public void onResume()
 		
 		if (m_wake_on_event)
 			doProcess(false);
+
+		for (int i = m_lifecycle_listeners.size() - 1; i >= 0; i--)
```

Comment:
This should happen before we `doProcess()` above I think - everything before that causes posts to the event queue, but that call could cause script to run which means all components should have been resumed.

Also this should loop from 0 to size so we resume earlier registered components before later ones.